ignite/pkg/cosmosgen/generate.go

```diff
@@ -109,19 +111,55 @@ func (g *generator) setup() (err error) {
 	return nil
 }
 
+func (g *generator) resolveDepencyInclude() (paths []string, err error) {
+	// Relative paths to proto directories
+	protoDirs := append([]string{g.protoDir}, g.o.includeDirs...)
+
+	// Create a list of proto import paths for the dependencies.
+	// These paths will be available to be imported from the chain app's proto files.
+	for rootPath, m := range g.thirdModules {
+		// Skip modules without proto files
+		if m == nil {
+			continue
+		}
+
+		// Check each one of the possible proto directory names for the
+		// current module and append them only when the directory exists.
+		for _, d := range protoDirs {
+			p := filepath.Join(rootPath, d)
+			f, err := os.Stat(p)
+			if err != nil {
+				if os.IsNotExist(err) {
+					continue
+				}
+
+				return nil, err
+			}
+
+			if f.IsDir() {
+				paths = append(paths, p)
+			}
+		}
+	}
+
+	return paths, nil
+}
+
 func (g *generator) resolveInclude(path string) (paths []string, err error) {
+	// Append chain app's proto paths
 	paths = append(paths, filepath.Join(path, g.protoDir))
 	for _, p := range g.o.includeDirs {
 		paths = append(paths, filepath.Join(path, p))
 	}
 
-	includePaths, err := protopath.ResolveDependencyPaths(g.ctx, g.cacheStorage, g.appPath, g.deps,
```

Comment:
`protopath.ResolveDependencyPaths` was also including global paths [here](https://github.com/ignite/cli/blob/develop/ignite/pkg/protopath/protopath.go#L44), and looks like it was applying some sort of filtering logic.

Are we sure to not have those?

Reply:
I will check it

Reply:
I just added the global `protoc` include paths which might be necessary if those import paths are being used. I just need to push it.

Im also removing the `protopath` package.

Regarding the filtering I think that it was making sure the module paths were relative if I recall correctly, but I will need some debugging to confirm that one.